### PR TITLE
Added host & play games function

### PR DESCRIPTION
**Pull request by deswra**

### CODE REVIEW

File: frontend/src/components/Quiz/QuizList/QuizList.jsx [hunk 1]
```diff
@@ -16,7 +17,13 @@ const QuizList = () => {
...
 
     return (
-        <Container>
+        <Container className="mt-3">
+            {quizzes.length === 0 ? (
+                <div className={`text-white ${styles.text}`}>
+                    <p>Wow, such empty!</p>
+                    <h2>It seems that you haven't created any quiz yet.</h2>
+                </div>
+            ) : null}
```
Comment (sonns) [inline frontend/src/components/Quiz/QuizList/QuizList.jsx:26]: You can use short-circuit operator for this

File: frontend/src/components/Play/Result/Result.jsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import Button from 'react-bootstrap/Button';
+import Row from 'react-bootstrap/Row';
+import Badge from 'react-bootstrap/Badge';
+import ListGroup from 'react-bootstrap/ListGroup';
+
+import { keyToVariants } from '../../../common/utils';
+import styles from './Result.module.css';
+
+const Result = ({ correctAnswers, leaderboard, playerNum, onNext, isEnd }) => {
+    console.log(leaderboard, playerNum);
```
Comment (sonns) [inline frontend/src/components/Play/Result/Result.jsx:13]: Please remove this line when you merge it into the main branch

File: frontend/src/components/Quiz/QuizList/QuizList.jsx (hunk 1 above)
Comment (deswra) [reply frontend/src/components/Quiz/QuizList/QuizList.jsx:26]: I'll try to use it in the future :grinning:

File: frontend/src/components/Play/Result/Result.jsx (hunk 2 above)
Comment (deswra) [reply frontend/src/components/Play/Result/Result.jsx:13]: Yes, I'll remember to remove all logging lines before merging with the master branch.

**Pull request merged**